test/rabbit_auth_backend_oauth2_test_util.erl

```diff
@@ -50,9 +50,16 @@ expirable_token() ->
     TokenPayload = fixture_token(),
     TokenPayload#{<<"exp">> := os:system_time(seconds) + timer:seconds(?EXPIRATION_TIME)}.
 
+expirable_token(Seconds) ->
```

Comment:
I can't see where this function is used, I guess it can be deleted. Note it should not add `timer:seconds(Seconds)`, just `Seconds`.